### PR TITLE
hw 17 for review v2.0

### PR DESCRIPTION
**Pull request by dezuz**

### CODE REVIEW

File: src/main/java/com/mateacademy/Poushkin.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+import org.apache.log4j.Logger;
+import org.springframework.context.annotation.Scope;
+import org.springframework.stereotype.Component;
+
+import javax.annotation.PostConstruct;
+import javax.annotation.PreDestroy;
+
+@Component("LyricistBean1")
+@Scope
```
Comment (Anderwerty) [inline src/main/java/com/mateacademy/Poushkin.java]: You don't need this annotation here.

File: pom.xml [hunk 2]
```diff
@@ -3,10 +3,50 @@
...
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.springframework.boot</groupId>
+                <artifactId>spring-boot-maven-plugin</artifactId>
+            </plugin>
+        </plugins>
+    </build>
 
 </project>
```
Comment (Anderwerty) [inline pom.xml]: Add a blank line at the end of the file.

**Review by Anderwerty — CHANGES_REQUESTED** [review]
Please put files in packages in which they should be.
Not all classes in the root package.

File: src/main/java/com/mateacademy/main/Start.java [hunk 3]
```diff
@@ -0,0 +1,33 @@
+package com.mateacademy.main;
```
Comment (Anderwerty) [inline src/main/java/com/mateacademy/main/Start.java]: It is better to leave this class in the root package.
com.mateacademy.main-> com.mateacademy

File: pom.xml [hunk 4]
```diff
@@ -3,10 +3,50 @@
...
     <groupId>com.mateacademy.spring-hw</groupId>
     <artifactId>spring-hw</artifactId>
     <version>1.0-SNAPSHOT</version>
 
+    <properties>
+        <java.version>1.8</java.version>
+    </properties>
+
+    <dependencies>
+        <dependency>
```
Comment (Anderwerty) [inline pom.xml:21]: It seems that you don't use this dependency. 
Am I right?
If so please remove it or use.

File: pom.xml [hunk 5]
```diff
@@ -3,10 +3,50 @@
...
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
+            <optional>true</optional>
+        </dependency>
+        <dependency>
+            <groupId>log4j</groupId>
+            <artifactId>log4j</artifactId>
+            <version>1.2.17</version>
+        </dependency>
+        <dependency>
```
Comment (Anderwerty) [inline pom.xml]: The same about this dependency.

File: pom.xml [hunk 6]
```diff
@@ -3,10 +3,50 @@
...
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-test</artifactId>
+            <scope>test</scope>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
```
Comment (Anderwerty) [inline pom.xml]: And what about this plugin. Do you need it?

**Review by Anderwerty — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 4 above)
Comment (dezuz) [reply pom.xml:21]: This dependency is used for ConfigurationApplicationContext in Start.java

**Review by Anderwerty — APPROVED** [review]
Good job.

**Pull request merged**